repo/packages/M/mr-redis/0/config.json

```diff
@@ -0,0 +1,63 @@
+{
+  "properties": {
+    "mr-redis": {
+      "description": "mr-redis specific configuration properties",
+      "properties": {
+        "dns-suffix": {
```

Comment:
mr-redis.marathon.mesos DNS suffix is used by the executors , removing this would be a problem.